R/connect.R

```diff
@@ -99,7 +103,18 @@ setMethod("dbConnect", "MariaDBDriver",
       bigint = bigint
     )
 
-    dbExecute(conn, "SET time_zone = '+00:00'")
+    if(exists('timezone')) {
```

Comment:
Can you please add a `timezone = "+0000"` argument to the function's formals (inside `function()`)? Then you can test with

```suggestion
    if (!missing("timezone")) {
```

and also avoid one `dbExecute()` call below.

Reply:
I removed the `dbExecute(conn, "SET autocommit = 0")`, but are you sure that it shouldn't be there?